app/assets/javascripts/discourse/app/components/sidebar/common/custom-section.hbs

```diff
@@ -8,13 +8,14 @@
   @class={{this.section.dragCss}}
 >
   {{#each this.section.links as |link|}}
-    {{#if link.external}}
+    {{#if (or link.external link.fullReload)}}
```

Comment:
I think I'll be better to move this logic into the `SectionLink` class instead of duplicating it in two places.